### DISCUSSION TITLE
Weird rendering in default themes using CaskaydiaCove nerd font.

### DISCUSSION THREAD

**Discussion by masonchester** [JanDeDobbeleer/oh-my-posh#4331] [Troubleshoot]
I am having weird rendering issues in the default theme, I tried cappuccin and the issue went away. I am on MacOS Sonoma using iTerm2 any suggestions would be appreciated.
<img width="727" alt="Screenshot 2023-10-10 at 2 21 14 PM" src="https://github.com/JanDeDobbeleer/oh-my-posh/assets/110614975/49d5327b-c319-428c-8763-adcd8f717129">

**Comment by JanDeDobbeleer** [comment]
@masonchester which weird rendering are you looking at? if it's the underline, that's a hyperlink.
